docs/index.asciidoc

```diff
@@ -24,6 +24,9 @@ Execute ruby code. This filter accepts inline ruby code or a ruby file.
 The two options are mutually exclusive and have slightly different ways of working,
 which are described below.
 
+NOTE: This plugin's concurrency-safety depends on your code. Be sure to read up on <<plugins-{type}s-{plugin}-concurrency,how to avoid concurrency issues>>.
+
+[id="plugins-{type}s-{plugin}-using-inline-script"]
 ===== Inline ruby code
 
 To inline ruby in your filter, place all code in the `code` option. This code will be executed for every event the filter receives. You can also place ruby code in the `init` option - it will be executed only once during the plugin's register phase.
```

Comment:
```suggestion
To add inline ruby in your filter, place all code in the `code` option. This code will be executed for every event the filter receives. You can also place ruby code in the `init` option. It will be executed only once during the plugin's register phase.
```